install

```diff
@@ -23,23 +27,27 @@ echo ">>> Starting installation on ARM device compatible with $arm_type"
 
 # Download and install IPFS
 ipfs_arch=${2-"linux-arm"}
-ipfs_version=${1-"v0.4.13"}
+ipfs_version=$(curl -s https://dist.ipfs.io/go-ipfs/versions | tail -1)
+ipfs_version=${1-$ipfs_version}
+ipfs_version="v${ipfs_version#v}"
```

Comment:
Do we really need this?

Reply:
I added this since I was very confused at the first times I specified a version as argument. For me it feelt natural to specify it as 
`./install "0.4.14"`
but expected is
`./install "v0.4.14"`
I see that both ways may be seen as the "natural" way of dooing it, so I added this to allow both formats. I think this is best UX wise.